addons/mail/static/tests/chatter_tests.js

```diff
@@ -520,6 +520,148 @@ QUnit.test('attachmentBox basic rendering', async function (assert) {
     form.destroy();
 });
 
+QUnit.test('attachmentBox dropzone available when hovering file [REQUIRE NON-INCOGNITO WINDOW]', async function (assert) {
+    assert.expect(7);
+
+    const form = await createView({
+        View: FormView,
+        model: 'partner',
+        data: this.data,
+        services: this.services,
+        arch: `<form string="Partners">
+                <sheet>
+                    <field name="foo"/>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_ids" widget="mail_thread"/>
+                </div>
+            </form>`,
+        res_id: 7,
+    });
+
+    var $button = form.$('.o_chatter_button_attachment');
+    assert.strictEqual($button.length, 1, "should have one attachment button");
+
+    await testUtils.dom.click($button);
+    assert.containsOnce(form, '.o_attachments_previews',
+        "should display some attachments on the attachments box");
+    assert.containsOnce(form, '.o_attachments_file_drop_zone',
+        "should have a dropzone to drag-and-drop files");
+    var $dropZoneContainer = form.$('.o_attachments_file_drop_zone');
+    assert.isNotVisible($dropZoneContainer, "dropzone should not be visible");
+
+    // hide AttachmentBox and test dragging file will automatically open it
+    await testUtils.dom.click($button);
+    await testUtils.nextTick();
```

Comment:
useless `nextTick()`, since there's already a next tick in `testUtils.dom.click()`